navigator/backend/src/main/scala/com/digitalasset/navigator/console/Pretty.scala

```diff
@@ -235,6 +235,10 @@ object Pretty {
     case model.ApiUnit() => PrettyPrimitive("<unit>")
     case model.ApiOptional(None) => PrettyPrimitive("<none>")
     case model.ApiOptional(Some(v)) => PrettyObject(PrettyField("value", argument(v)))
+    case model.ApiMap(map) =>
+      PrettyArray(map.toList.map {
```

Comment:
This outputs a list of objects, each with a single key-value pair. IMO it would be better to output a single object with several key-value pairs.

```suggestion
          PrettyObject(map.toList.map {case (key, value) => PrettyField(key, argument(value)) })
```